experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts

```diff
@@ -39,7 +39,7 @@ export class Bubblebench extends DataObject {
 			throw new Error("hasInitialized called but tree is still undefined");
 		}
 		this.maybeAppState = new AppState(
-			this.tree.root as ClientTreeProxy[] & EditableField,
+			this.tree.root as ClientsField,
```

Comment:
Future: going back to your idea of passing the schema to the factory, we could then return a SharedTree<T>, where T is used to correctly type .root.

Reply:
yes, once our typed schema APIs are fancy enough to handle it.